Formula/cairomm@1.14.rb

```diff
@@ -7,7 +7,7 @@ class CairommAT114 < Formula
 
   livecheck do
     url "https://cairographics.org/releases/?C=M&O=D"
-    regex(/href=.*?cairomm[._-]v?(1\.14\.\d*[02468](?:\.\d+)*)\.t/i)
+    regex(/href=.*?cairomm[._-]v?(1\.14(?:\.\d+)*)\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?cairomm[._-]v?(1\.14(?:\.\d+)+)\.t/i)
```
maybe?

Reply:
Done, didn't change that because it's the same as `cairomm` but it seems like `+` may work there too.